bftengine/src/bftengine/messages/PrePrepareMsg.cpp

```diff
@@ -173,6 +175,7 @@ PrePrepareMsg::PrePrepareMsg(ReplicaId sender,
   b()->viewNum = v;
   b()->time = 0;
 
+  // b()->time = ready ? *((int64_t*)(spanContext.data().c_str() + 6 + 8 + 8 + 8 + 2)) : 0;
```

Comment:
Is this comment intentionally added here? If this is the case can we add some explanation as well?

Reply:
Removed